build.gradle

```diff
@@ -129,7 +128,6 @@ project(':ambry-server') {
     dependencies {
         compile project(':ambry-clustermap'),
                 project(':ambry-messageformat'),
-                project(':ambry-metrics'),
```

Comment:
The ambry-metrics project is being removed in this commit, right? Or is there something I forgot?